templates/resources/device_posture_rule.md

```diff
@@ -19,10 +19,14 @@ resource "cloudflare_device_posture_rule" "corporate_devices_posture_rule" {
   schedule    = "24h"
   expiration  = "24h"
   match {
-    platform = "mac"
+    platform = "linux"
   }
   input {
     id = cloudflare_teams_list.corporate_devices.id
+    version            = 1.0.0
+    operator           = <
+    os_distro_name     = ubuntu
+    os_distro_revision = 1.0.0
```

Comment:
these fields aren't present in the schema or public API docs; so we won't be able to add them at this time.